filters/auth/oidc_test.go

```diff
@@ -803,8 +844,10 @@ func TestOIDCSetup(t *testing.T) {
 				reqURL.RawQuery = q.Encode()
 			}
 
-			if tc.upstreamheaders != "" {
-				sargs = append(sargs, tc.upstreamheaders)
+			sargs = append(sargs, tc.upstreamheaders)
```

Comment:
This will append empty value unconditionally. The code seems to check for empty value so this would work. 

For the consistency lets remove `if tc.subdomainsToRemove != ""` check as well as your code also checks for empty value.